include/field.hxx

```diff
@@ -505,22 +505,22 @@ inline T pow(BoutReal lhs, const T& rhs, REGION rgn) {
 /// This loops over the entire domain, including guard/boundary cells by
 /// default (can be changed using the \p rgn argument).
 /// If CHECK >= 3 then the result will be checked for non-finite numbers
-FIELD_FUNC(sqrt, ::sqrt);
+FIELD_FUNC(sqrt, ::sqrt)
```

Comment:
I think I may have put in a lot of these semicolons after macro calls. I think it was due to the editor indentation; without the semicolon my editor at the time would think the next line was a continuation, and helpfully indent it. Is this an issue for clang formatting?

Reply:
Yeah, I seem to recall having problems here in the past, but both emacs and clang-format Do The Right Thing now, so I'm happy removing them.